### DISCUSSION TITLE
Import contacts error

### DISCUSSION THREAD

**Discussion by jekv2** [FossifyOrg/General-Discussion#655] [General]
![Screenshot_20240117-142100](https://github.com/FossifyOrg/Contacts/assets/7735249/ff1e6f74-eed8-4700-8ffd-d06781a5f7ac)

**Comment by Aga-C** [comment]
Don't create new discussions. Please raise the bug as I described in your previous topic: https://github.com/FossifyOrg/Contacts/discussions/24#discussioncomment-8161557.
